src/ImageSharp/Processing/KnownFilterMatrices.cs

```diff
@@ -432,6 +432,30 @@ public static ColorMatrix CreateSaturateFilter(float amount)
             return m;
         }
 
+        /// <summary>
+        /// Create a lightness filter matrix using the given amount.
+        /// </summary>
+        /// <remarks>
+        /// A value of 0 will create an image that is completely black. A value of 1 makes the image completely white.
+        /// Other values are linear multipliers on the effect. Values of an amount over 1 are allowed, providing brighter results.
+        /// </remarks>
+        /// <param name="amount">The proportion of the conversion. Must be greater than or equal to 0.</param>
+        /// <returns>The <see cref="ColorMatrix"/></returns>
+        public static ColorMatrix CreateLightnessFilter(float amount)
+        {
+            Guard.MustBeBetweenOrEqualTo(amount, 0, 1F, nameof(amount));
+            return new ColorMatrix
```

Comment:
We're going to need some tests for this. I recommend copying the brightness ones.

Reply:
I'll try to do it.